src/logger.js

```diff
@@ -17,4 +17,8 @@ export default class Logger {
 	static log( ...args ) {
 		console.log( ...args );
 	}
+
+	static warn( ...args ) {
```

Comment:
Do we achieve something using the `Logger` class instead of direct `console` calls?

Reply:
TBH it was so long time I don't remember :D

Reply:
Maybe for easier stubbing? I'll create an issue for it as it will be very easy to get rid of it 😄

Reply:
I extracted it to https://github.com/ckeditor/ckeditor5-inspector/issues/63.